<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="serviceLabel">
       <property name="text">
        <string>Service:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="serviceComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="serviceURLLabel">
       <property name="text">
        <string>Service URL:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="serviceURLLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="userIDLabel">
       <property name="text">
        <string>User/ID:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="userIDLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="passwordKeyTokenLabel">
       <property name="text">
        <string>Password/Key/Token:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="passwordKeyTokenLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="presetLabel">
       <property name="text">
        <string>Preset:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="presetComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonCreatePreset">
       <property name="text">
        <string>Create Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonUpdatePreset">
       <property name="text">
        <string>Update Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDeletePreset">
       <property name="text">
        <string>Delete Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
